### ISSUE TITLE
[Question] Are any of these features in scope

### ISSUE THREAD

**Issue by patcon** [djc/awmy#23]
Hi there! This is a really awesome little tool!

I've build a small landing page for entry into our own meetings, to help make ensure folks in [our community](envirodatagov.org) know what they are consenting to on arrival at a video meeting:
https://edgi-video-call-landing-page.herokuapp.com (click the sample link to see the page itself)

It's nothing fancy (which is good). But it's unfortunately very single-purpose. I was wondering whether you might be open to discussion on add two features to yours that would allow us to start using the same codebase :)

As I see it, two thing would works:
- a linked button to enter the meeting
- a place for description, accepting markdown

Curious your thoughts. Thanks again for the awesome tool!

**Comment by djc** [comment]
Hey, thanks for the kind words!

As you may or may not know, AWMY is completely stateless: all the information on a page is contained in the URL that you browse to. I don't really want to change that, either: I think the way it works is exactly right. This probably makes your second request a bit harder to do in a nice way, though.

On the other hand, the first request could probably be done, by having an optional query argument that passes in a URL for the button to point to. That would still make the URLs substantially longer, though, and therefore harder to read/follow/understand.

**Comment by patcon** [comment]
ah ok, gotcha! What about compressing a longer optional description into base64? Yes, less terse urls for those who use them, but it the flags are optional...? :)

I mean, I can always fork, it's just less exciting to be working on something on my own

**Comment by djc** [comment]
Yeah, that's probably the best way forward. Do you want to implement something like this yourself?
